### ISSUE TITLE
when release ckpt of the reward model and training dataset?

### ISSUE THREAD

**Issue by katherine-bf** [salesforce/HIVE#5]

**Comment by katherine-bf** [comment]
hello, thanks for your excellent work!
when you release ckpt of the reward model or is there any planning to release the training dataset

**Comment by shugerdou** [comment]
@katherine-bf Thanks for your interest. We will release everything including the model and the dataset in about a week. Sorry that we are busy with some deadlines.

**Comment by distillation-dcf** [comment]
When will you release your 1.32M training dataset? It would really help to reimplement your work!

**Comment by shugerdou** [comment]
@katherine-bf  @distillation-dcf  Sorry that we are delayed due to some other projects. We have released the first version of training data. Please pull the latest code. We will release the reward model soon. 
@distillation-dcf  Please note that is our training data was reported as 1M in the paper. The released training data is about 1.1M. What you mentioned (1.32M) adds the data from instructpix2pix as well.
